tools/vms-generator/utils/utils.go

```diff
@@ -195,15 +196,18 @@ func addRNG(spec *v1.VirtualMachineInstanceSpec) *v1.VirtualMachineInstanceSpec
 }
 
 func addContainerDisk(spec *v1.VirtualMachineInstanceSpec, image string, bus v1.DiskBus) *v1.VirtualMachineInstanceSpec {
-	disk := &v1.Disk{
-		Name: "containerdisk",
-		DiskDevice: v1.DiskDevice{
-			Disk: &v1.DiskTarget{
-				Bus: bus,
+	// Only add a reference to the disk if it isn't using the default v1.DiskBusSATA bus
```

Comment:
Can you clarify why this is necessary? Is it because you count on mutating webhooks to set this disk as a default?

Reply:
Yes, it's a little heavy handed but it's really the only way of demonstrating this behaviour in our examples at the moment without flavors and preferences etc.